builtin/credential/approle/path_tidy_user_id.go

```diff
@@ -39,206 +40,224 @@ func (b *backend) tidySecretID(ctx context.Context, req *logical.Request) (*logi
 		return resp, nil
 	}
 
-	s := req.Storage
+	go b.tidySecretIDbackground(req.Storage)
 
-	go func() {
-		defer atomic.StoreUint32(b.tidySecretIDCASGuard, 0)
+	resp := &logical.Response{}
+	resp.AddWarning("Tidy operation successfully started. Any information from the operation will be printed to Vault's server logs.")
```

Comment:
Here in this warning, we could indicate that the process happens in background. That might help some people understand that it is an ongoing process and not yet completed.

Reply:
I hesitate to mess with the existing warning message in case anyone is already parsing for it.

Reply:
Fair enough